lib/logstash/inputs/lumberjack.rb

```diff
@@ -126,4 +127,69 @@ def start_buffer_broker(output_queue)
       end
     end
   end
+
+  class Decoration < LogStash::Inputs::Base
+    public
+    def decorate(event)
+      super(event)
+    end
+  end # class Decoration
+
+  class Client
```

Comment:
renamed Client to ConnectionDecorator (naming Client was too misleading). With the plugins module structure and ConnectionDecorator, EventDecoration being inner classes I refrained from moving them into separate files... kinda painfull.

Any importance (semantically) in module/class naming of plugins for logstash? Maybe we can restructure to

    module LogStash::Input::Lumberjack
        class Plugin < LogStash::Inputs::Base ...
        class ConnectionDecorator ...
        class EventDecoration ...
    end